main/templates/main/layout.html

```diff
@@ -7,7 +7,8 @@
         <title>{% block title %}{% endblock %}</title>
 
         {% if not request.subdomain %}
-        <meta name="description" content="Blogging platform for minimalists.">
+        {# TODO: how to add it here? #}
```

Comment:
This is the only place I couldn't figure how to add the description... Would you be able to help me here, please?

Reply:
Sure! Add it on the request somewhere here: https://github.com/mataroa-blog/mataroa/blob/fbb6ef628935af95a03a70a28e176e588418c2da/main/middleware.py#L39

Reply:
Sure! Add it on the request somewhere here: https://github.com/mataroa-blog/mataroa/blob/fbb6ef628935af95a03a70a28e176e588418c2da/main/middleware.py#L39